content/docs/installation/operator-lifecycle-manager.md

```diff
@@ -41,7 +41,7 @@ from the [Krew Kubectl plugins index][] and then use that to install the cert-ma
 ```sh
 operator-sdk olm install
 kubectl krew install operator
-kubectl operator install cert-manager -n operators --channel stable --approval Automatic
+kubectl operator install cert-manager -n cert-manager --channel candidate --approval Automatic --create-operator-group
```

Comment:
This didn't work for me without first creating the cert-manage ns.
And we should continue to recommend the `stable` channel.
The `candidate` channel contains test packages. 


```bash
kubectl create ns cert-manager
kubectl operator install cert-manager -n operators --channel stable --approval Automatic --create-operator-group -n cert-manager
```

```
operatorgroup "cert-manager" created
subscription "cert-manager" created
operator "cert-manager" installed; installed csv is "cert-manager.v1.11.4"
```

Reply:
Excellent points. Late night drive-by docs PRs to get stuff installed in the homelab 😅